minlora/model.py

```diff
@@ -85,6 +85,13 @@ def apply_lora(layer, register=True, merge=False, lora_config=default_lora_confi
                 parametrize.remove_parametrizations(layer, attr_name, leave_parametrized=merge)
 
 
+def apply_lora_by_name(model, target_module_names, register=True, merge=False, lora_config=default_lora_config):
```

Comment:
This function is an extension to `add_lora`, so I think it's better to name it `add_lora_by_name` and put it under `add_lora`.